frontend/public/components/pod.jsx

```diff
@@ -189,7 +189,7 @@ export const PodDetailsList = ({pod}) => {
 };
 
 export const PodResourceSummary = ({pod}) => (
-  <ResourceSummary resource={pod} showPodSelector={false} showNodeSelector={false}>
+  <ResourceSummary resource={pod}>
```

Comment:
Pods can have a node selector

```suggestion
  <ResourceSummary resource={pod} showNodeSelector>
```